src/webtoon_downloader.py

```diff
@@ -463,14 +567,42 @@ def slugify_file_name(file_name: str) -> str:
     return re.sub(r"[^\w.-]", "", file_name.strip().replace(" ", "_"))
 
 
+def get_chapter_dir(
+    chapter_info: ChapterInfo,
+    zeros: int,
+    separate_chapters: bool,
+    export_texts: bool
+) -> str:
+    """
+    Get the relative directory to use for a chapter, given the supplied options.
+
+    Arguments:
+        chapter_info:      Information about this chapter
+        zeros:             Number of digits to use for the chapter-number
+        separate_chapters: Selector if each chapter should be its own directory
+        export_texts:      Selector if the chapter-title should be part of the
+                           directory name
+
+    Returns:
+        Relative directory for files in chapter
+    """
+    if not separate_chapters:
+        return '.'
+    dirname = f"{chapter_info.chapter_number:0{zeros}d}"
+    if export_texts:
+        dirname += f"_{slugify_file_name(chapter_info.title)}"
```

Comment:
I feel like `export_texts` should not impact the output file from `separate`. I agree that this should be better. But I feel that this should be removed now and possibly worked on in the future for a better solution for https://github.com/Zehina/Webtoon-Downloader/issues/3